model/project_parser.go

```diff
@@ -439,7 +439,7 @@ func LoadProjectFromVersion(v *Version, identifier string, shouldSave bool) (*Pr
 				"config_number": v.ConfigUpdateNumber,
 				"message":       "error updating version's project",
 			}))
-			return nil, errors.Wrap(err, "error updating version with project")
+			return p, nil
```

Comment:
I'm on the fence about swallowing an error here, because the danger is that some caller of this function could continue, assuming that it accomplished all his work. Is is necessary to not return an error?

Reply:
It's not necessarily necessary to not return an error, but in the case there's a database error or something updating the version we still do have the parser project which is what we want. This would swallow issues with updating the version though and is a splunk error we may want to keep an eye on. I'm fine with either.

Reply:
Let's return the error.